### PR TITLE
Fixed MP3 detection.

### PR DESCRIPTION
**Pull request by IceReaper**
When adding MP3 to the audio formats in mod yaml, everything listed afterwards will break, as the mp3 implementation does not check for the mp3 headers. This PR fixes this, so the mp3 loader wont be used for non-mp3 files, breaking following formats.

### CODE REVIEW

File: OpenRA.Mods.Common/AudioLoaders/Mp3Loader.cs [hunk 1]
```diff
@@ -18,12 +18,37 @@ namespace OpenRA.Mods.Common.AudioLoaders
 {
 	public class Mp3Loader : ISoundLoader
 	{
+		bool IsMp3(Stream s)
```
Comment (PunkPun) [inline OpenRA.Mods.Common/AudioLoaders/Mp3Loader.cs:21]:
```suggestion
		bool IsMP3(Stream s)
```

Isn't it abbreviation?

Comment (IceReaper) [reply OpenRA.Mods.Common/AudioLoaders/Mp3Loader.cs:21]: But we use "Mp3" everywhere?

**Review by abcdefg30 — APPROVED** [review]

**Review by RoosterDragon — APPROVED** [review]

**Comment by Mailaender** [comment]
Thanks for cleaning up after me.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d68774baabdf1757b936b88913ae3e35b15cd133...68641e166dd0812d383fb0c683b26938a6aeba73)

**Comment by Mailaender** [comment]
For stable?

**Comment by PunkPun** [comment]
I don't think it's necessary, but I wouldn't mind

**Comment by Mailaender** [comment]
https://github.com/OpenRA/OpenRA/commit/716f6939b91504cce986b24e0432cf6b75e40a76

**Pull request merged**